src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.props

```diff
@@ -126,4 +126,6 @@ Copyright (c) .NET Foundation. All rights reserved.
 
   <Import Project="$(MSBuildThisFileDirectory)Microsoft.NET.PackTool.props" />
   <Import Project="$(MSBuildThisFileDirectory)Microsoft.NET.PackProjectTool.props" />
+
+  <Import Project="$(MSBuildThisFileDirectory)../../Microsoft.NET.Sdk.WindowsDesktop/targets/Microsoft.NET.Sdk.WindowsDesktop.props"/>
```

Comment:
To avoid having to sync checkins between the wpf and sdk repos, you could update this .props file to set a `WindowsDesktopPropsImported` property to true.  Then when adding this import you could condition it on that property not being set.

Separately, what I would like to do eventually is to move the implicit WindowsDesktop FrameworkReferences to Windows-specific props in the SDK repo (which would eventually go in a Windows workload).  The rest of the logic in the WindowsDesktop props/targets is specific to WPF, so those targets could be renamed to Microsoft.WPF.props/.targets or similar.

Reply:
Without their side of change I am importing it twice. I need to somehow detect the props already been imported. And I see a prop in [there](https://github.com/dotnet/wpf/blob/52b9067325cd90cc38e100584075abe5a8489139/packaging/Microsoft.NET.Sdk.WindowsDesktop/targets/Microsoft.NET.Sdk.WindowsDesktop.props) can indicate that